### PR TITLE
Update to support clappr 0.3

### LINKED ISSUE #30: Needs updating to support clappr 0.3+

### PR DESCRIPTION
**Pull request by ziddey**
Use new activePlayback/activeContainer/CORE_ACTIVE_CONTAINER_CHANGED.
Note this breaks compatibility with older versions of clappr.

Fixes #30

### CODE REVIEW

**Review by leandromoreira — APPROVED** [review]

File: src/chromecast.js [hunk 1]
```diff
@@ -261,6 +261,7 @@ export default class ChromecastPlugin extends UICorePlugin {
     let mimeType = ChromecastPlugin.mimeTypeFor(src)
     let mediaInfo = new chrome.cast.media.MediaInfo(src)
     mediaInfo.contentType = this.options.contentType || mimeType
+    mediaInfo.customData = this.options.customData
```
Comment (kslimani) [inline src/chromecast.js:264]: @ziddey : maybe also edit the readme file to document/explain this new `customData` option ?

Comment (ziddey) [reply src/chromecast.js:264]: Ack, didn't mean to sneak in that commit. Pushed changes to README.md

**Review by leandromoreira — APPROVED** [review]

**Review by towerz — APPROVED** [review]

**Comment by towerz** [comment]
Thanks, @ziddey! 🍻 

I think we need to check why some of those plugins are not working with 0.3.0 as they are. There should be no compatibility issues, only some internal implementations should break. Perhaps some of the events haven't been mapped to their previous versions when triggered. This version was supposed to have redundant events to allow us to work on version/compatibility checks for our next version (0.4.0) which would introduce those breaking changes.

**Comment by ziddey** [comment]
The only line that _needed_ to be changed was
`    this.listenTo(this.core.mediaControl, Events.MEDIACONTROL_CONTAINERCHANGED, this.containerChanged)	`
to
`    this.listenTo(this.core, Events.CORE_ACTIVE_CONTAINER_CHANGED, this.containerChanged)
`
If I remember right, the old event would still fire, but not enough times (when it actually needed to fire).

The other functions are still working as expected, but I updated them to the new spec.

**Pull request merged**